x-pack/plugins/observability_solution/observability/server/routes/register_routes.ts

```diff
@@ -88,16 +87,6 @@ export function registerRoutes({ config, repository, core, logger, dependencies
 
           return response.ok({ body: data });
         } catch (error) {
-          if (error instanceof ObservabilityError) {
-            logger.error(error.message);
-            return response.customError({
-              statusCode: getHTTPResponseCode(error),
-              body: {
-                message: error.message,
-              },
-            });
-          }
-
```

Comment:
This was specific to SLO. Since we moved SLO outside of observability/ this handler is not use anymore